docs/reference/ml/anomaly-detection/categories.asciidoc

```diff
@@ -1,6 +1,28 @@
 [role="xpack"]
 [[ml-configuring-categories]]
-=== Categorizing log messages
+=== Configuring categorization
```

Comment:
To fit with the other examples in this section, I think it's better to just stick with something like this:
```suggestion
=== Categorizing data
```